packages/mdc-textfield/README.md

```diff
@@ -426,6 +426,32 @@ Returns a boolean specifying whether or not the input is disabled.
 
 Updates the input's disabled state.
 
+##### MDCTextfieldFoundation.setValid(isValid: boolean)
+
+Sets the validity state of the Text Field. Triggers custom validity checking.
+
+##### MDCTextfieldFoundation.handleTextFieldInteraction(evt: !Event)
```

Comment:
Nit: FWIW, I'm pretty sure we follow typescript conventions when documenting APIs in READMEs, so the `!`s here might not make sense...

Reply:
Done.